README.md

```diff
@@ -27,9 +27,9 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖  React Hook Calculator logic<a name="about-project"></a>
+# 📖  React Hook Calculator with Quotes.<a name="about-project"></a>
```

Comment:
[REQUIRED] - Please kindly fix the broken links in your readme file

- [ ] The link to your `Twitter` and `LinkedIn` profiles. kindly fix that. 🛠👍

![image](https://user-images.githubusercontent.com/78251927/234691075-cb05b517-6889-463c-a9d6-ba4badb1789c.png)

![image](https://user-images.githubusercontent.com/78251927/234691240-6c834f6e-2d48-4bdc-9b9e-e5bb1af967ff.png)

- [ ] Let's avoid broken links in our readme file for professional repositories. 💪🤗

![image](https://user-images.githubusercontent.com/78251927/234691326-299da03f-b1f0-4686-90da-99249613ec77.png)